cmd/daprd/components/wfbackend_sqlite.go

```diff
@@ -0,0 +1,24 @@
+//go:build allcomponents
```

Comment:
I wonder if all workflow backends should be built with "stablecomponents" too

Reply:
Not all components will be stable by default, but sqlite should stable I believe. I will mark it stable.